index.html

```
@@ -412,6 +412,22 @@
                     </div>
                 </div>
 
+ feature/india-ancient-dolmens-profile
+            <div class="nav-dropdown">
+                <button class="nav-link dropdown-toggle" aria-haspopup="true" aria-expanded="false">Culture ▾</button>
+                <div class="dropdown-menu">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div></div> ], start tag match failed [ <div class="dropdown-menu"> ] on line 418.

(tag-pair)